charts/fis-hmc-api/values.yaml

```diff
@@ -35,8 +35,8 @@ java:
     HMC_HOST: 'hmc-servicebus-{{ .Values.global.environment }}.servicebus.windows.net'
     APPLICATION_NAME: 'fis_hmc_api'
     MANAGE_CASE_REDIRECT_URL: 'https://manage-case.{{ .Values.global.environment }}.platform.hmcts.net/oauth2/callback'
-    HMC_CFT_HEARING_BASEURL: 'http://hmc-cft-hearing-service-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal/hearings/'
-    HMC_CFT_HEARING_BASEURL_FOR_FEIGN: 'http://hmc-cft-hearing-service-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal/'
+    HMC_CFT_HEARING_BASEURL: 'http://hmc-cft-hearing-service-demo.service.core-compute-demo.internal/hearings/'
```

Comment:
Env specific changes to be be managed by flux config

Reply:
Apologies forgot to revert back ... Thank you addressed the above @shashisk97